custom-routes-vis/src/main/webapp/script.js

```diff
@@ -23,15 +23,21 @@ const MarkerNames = {
   DESTINATION: 'destination',
 };
 
-// Array holding origin and destination markers.
-let originDestinationMarkers = []
-
-/** Creates the world map with markers. */
-function createWorldMapWithMarkers() {
-  map = new google.maps.Map(
-    document.getElementById('map'),
-    {center: {lat: 38.46049, lng: -5.428423}, zoom: 3});
-  
+let originDestinationMarkers = [];
```

Comment:
Rename this to tripMarkers or something like that. @lawrencezjh What do you think?